docs/site/src/app/services/partners.service.ts

```diff
@@ -0,0 +1,314 @@
+const countryList = {
```

Comment:
You should load this line from a JSON file that stored in the assets folder. We should fetch this information at runtime instead of including them into the bundle directly.